js/privatebin.js

```diff
@@ -4689,6 +4729,7 @@ jQuery.PrivateBin = (function($, RawDeflate) {
             TopNav.showEmailButton();
 
             TopNav.hideRawButton();
+	    TopNav.hideDownloadButton();
```

Comment:
Tabulation

Reply:
fixed in 853a4f386fa9a870cb2aa3616d43b61d0f10ec5f